library/ssl_tls.c

```diff
@@ -6545,7 +6545,7 @@ void mbedtls_ssl_session_free( mbedtls_ssl_session *session )
 
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
-    if( session->ticket_nonce_len > 0 ) mbedtls_free( session->ticket_nonce );
+    if( session->ticket_nonce != NULL ) mbedtls_free( session->ticket_nonce );
```

Comment:
This check doesn't seem to be necessary, since `free()` is guaranteed to be a no-op when passed a NULL pointer, if I remember correctly.

Reply:
Good point. I'll update this.